mesonbuild/interpreter/interpreter.py

```diff
@@ -2638,6 +2639,15 @@ def func_get_variable(self, node: mparser.BaseNode, args: T.Tuple[T.Union[str, D
     def func_is_variable(self, node: mparser.BaseNode, args: T.Tuple[str], kwargs: 'TYPE_kwargs') -> bool:
         return args[0] in self.variables
 
+    @typed_pos_args('unset_variable', str)
+    @noKwargs
```

Comment:
```python
@FeatureNew('unset_variable', '0.60.0')
```

Reply:
Whoops totally forgot about that. Will add it right now!